extensions/kubernetes-config/runtime/src/test/java/io/quarkus/kubernetes/client/runtime/SecretConfigSourceUtilTest.java

```diff
@@ -102,14 +102,18 @@ void testWithAllKindsOfData() {
         List<ConfigSource> configSources = sut.toConfigSources(secret.getMetadata().getName(), secret.getData());
 
         assertThat(configSources).hasSize(4);
-        assertThat(configSources).filteredOn(c -> c.getName().toLowerCase().contains("literal"))
-                .hasOnlyOneElementSatisfying(c -> {
-                    assertThat(c.getProperties()).containsOnly(entry("some.key", "someValue"));
-                });
-        assertThat(configSources).filteredOn(c -> c.getName().toLowerCase().contains("application.properties"))
```

Comment:
Why not keep the existing tests? They should still pass, no? At least the one that verifies content of `application.properties`.

Reply:
For some reason the existing code was adding a "-literal" to the secret key. No idea why, I searched the code and it wasn't removed so I removed it. It was on line 53